src/views/product-root-docs-path-landing/server.ts

```diff
@@ -84,12 +84,12 @@ const generateGetStaticProps = ({
 	pageContent,
 	product,
 	productSlugForLoader,
+	basePath,
 }: GenerateGetStaticPropsArguments) => {
-	const basePath = 'docs'
 	const currentRootDocsPath = product.rootDocsPaths.find(
 		(rootDocsPath: RootDocsPath) => rootDocsPath.path === basePath
 	)
-	const baseName = currentRootDocsPath.shortName
+	const baseName = currentRootDocsPath.shortName || currentRootDocsPath.name
```

Comment:
Fallback here allows us to not have to define duplicative `shortName` values. I think some of this could be DRY'd up by consolidating `rootDocsPath` config with `src/page`-level `basePath` / `baseName` config, but leaving that for a separate PR (may align with `[[...allDocs]].tsx` work in #705, for example).

Reply:
`name` is great to fall back to! 👍🏻